projects/aca-content/src/lib/components/details/details.component.ts

```diff
@@ -99,6 +99,22 @@ export class DetailsComponent extends PageComponent implements OnInit, OnDestroy
       .subscribe((aspectActions) => {
         this.aspectActions = aspectActions;
       });
+    this.infoDrawerOpened$
+      .pipe(
+        filter((opened) => !opened),
+        first(),
+        takeUntil(
```

Comment:
I think that you don't need `takeUntil` when you're using `first` as after emitting the first value `first` then completes according to rxjs docs

Reply:
Yes but what in case if somebody will exit page before any value will be emitted? Then first won't be executed but still we would need to cancel subscription

Reply:
Right good point since `infoDrawerOpened$` comes from the store so it won't complete when user will exit the page